### ISSUE TITLE
 Chronograf (1.7.13) Cannot read property 'split' of null

### ISSUE THREAD

**Issue by ocalvo** [influxdata/chronograf#5274]
What browser are you using (name and version)?
Edge Canary Build

What operating system are you using?
Windows 10 19H1 (1903)

Please describe what you were trying to do when you encountered this error:
Installing InfluxDB on Hassio

```
TypeError: Cannot read property 'split' of null
    at http://192.168.1.102:8123/api/hassio_ingress/OrVpEfWbaBmiAqD7ZWrwQO5312uOhAHJdHxOv3Ks7Z0//src.b085f594.js:3396:2294
    at Array.filter (<anonymous>)
    at Object.s.reset [as onResetTimeMachine] (http://192.168.1.102:8123/api/hassio_ingress/OrVpEfWbaBmiAqD7ZWrwQO5312uOhAHJdHxOv3Ks7Z0//src.b085f594.js:3396:2217)
    at d.c (http://192.168.1.102:8123/api/hassio_ingress/OrVpEfWbaBmiAqD7ZWrwQO5312uOhAHJdHxOv3Ks7Z0//src.b085f594.js:4118:3369)
    at new d (http://192.168.1.102:8123/api/hassio_ingress/OrVpEfWbaBmiAqD7ZWrwQO5312uOhAHJdHxOv3Ks7Z0//src.b085f594.js:1911:1841)
    at Tl (http://192.168.1.102:8123/api/hassio_ingress/OrVpEfWbaBmiAqD7ZWrwQO5312uOhAHJdHxOv3Ks7Z0//src.b085f594.js:717:48187)
    at ja (http://192.168.1.102:8123/api/hassio_ingress/OrVpEfWbaBmiAqD7ZWrwQO5312uOhAHJdHxOv3Ks7Z0//src.b085f594.js:717:63002)
    at $a (http://192.168.1.102:8123/api/hassio_ingress/OrVpEfWbaBmiAqD7ZWrwQO5312uOhAHJdHxOv3Ks7Z0//src.b085f594.js:717:69008)
    at oo (http://192.168.1.102:8123/api/hassio_ingress/OrVpEfWbaBmiAqD7ZWrwQO5312uOhAHJdHxOv3Ks7Z0//src.b085f594.js:717:91055)
    at uo (http://192.168.1.102:8123/api/hassio_ingress/OrVpEfWbaBmiAqD7ZWrwQO5312uOhAHJdHxOv3Ks7Z0//src.b085f594.js:717:91442)
```

**Comment by cgwyllie** [comment]
This looks like a duplicate of these issues: https://github.com/influxdata/chronograf/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aclosed+split+of+null

Only difference appears to be browser.

According to https://github.com/influxdata/chronograf/issues/5271 it may be resolved in 1.7.14.

**Comment by ocalvo** [comment]
No longer repro
